cypress/e2e/userList.cy.ts

```diff
@@ -0,0 +1,176 @@
+import { Role } from '@prisma/client';
+
+const adminUser = {
+  name: 'Anna Admin',
+  email: 'anna@traininghub.org',
+  role: Role.ADMIN,
+  profileCompleted: true,
+};
```

Comment:
```suggestion
const adminUser = {
  name: 'Anna Admin',
  email: 'anna@traininghub.org',
  role: Role.ADMIN,
};
```
No need for this line